### PR TITLE
fix(frontend): remove the sidebar on Stats page

### PR DESCRIPTION
**Pull request by icerove**
delete sidebar for now

### CODE REVIEW

**Comment by frol** [comment]
**TL;DR: remove the navigation.**

Let's admit that the page is not optimized for mobile at all, including the charts.

Even more, the page is not optimized for the desktop if you make the page take half of the screen:

image

I don't like the proposed button floating in the middle of the screen:

image

I suggest we don't try to make it worse.

**Comment by icerove** [comment]
I will ask Corwin's help here. @frol

**Comment by icerove** [comment]
I delete it for now. @frol

File: frontend/public/static/images/icon-sidebar.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 0 2 1.69h.3a2 2 0 0 0 1.68-2.28l-9.61-62.28a2 2 0 0 0 -2.24-1.72zm10.6 82.35 1.37 8.89a2 2 0 0 0 2 1.7h.31a2 2 0 0 0 1.67-2.28l-1.43-8.97a2 2 0 0 0 -4 .61zm18.4 16.65h-2.92a1.91 1.91 0 0 0 .06-.86l-16.37-106.75a2 2 0 0 0 -2.28-1.7l-16.64 2.57a2 2 0 0 0 -1.85-1.26h-18a2 2 0 0 0 -2 2v106h-1v-106a2 2 0 0 0 -2-2h-36a2 2 0 0 0 -2 2v106h-7a2 2 0 0 0 0 4h84.59a1.85 1.85 0 0 0 .78.17h.3l.94-.15h21.39a2 2 0 0 0 0-4zm-65 0h-32v-104h32zm23 0h-14v-104h14zm4 0v-84.87l13 84.87zm17-.11-15.72-102.8 13.84-2.12 15.76 102.8zm-60-44.89a2 2 0 0 0 -2 2v27a2 2 0 0 0 4 0v-27a2 2 0 0 0 -2-2zm-8.75-35.06.07.11.05.07a2 2 0 0 0 .22.27h.05a2.11 2.11 0 0 0 .23.19l.07.05a3.07 3.07 0 0 0 .31.17l.3.1h.1a2.22 2.22 0 0 0 .37 0 2.31 2.31 0 0 0 .46 0l.14-.05a1.82 1.82 0 0 0 .3-.11 2.2 2.2 0 0 0 .62-.48l4-5.53-4.58 2v-9.58l10.97 13.11a2.05 2.05 0 0 0 1.58.74h.43a2.05 2.05 0 0 0 2.06-2v-13.94a2 2 0 0 0 -.32-1.09 2 2 0 0 0 -.3-.36l-.06-.06a2.86 2.86 0 0 0 -.3-.22l-.13-.07a1.79 1.79 0 0 0 -.32-.13h-.13a2 2 0 0 0 -.49-.07 1.84 1.84 0 0 0 -.47.06h-.13l-.3.11a2 2 0 0 0 -.61.47l-3.48 5.66 4.04-2.28v9.78l-10.93-13.12a2.05 2.05 0 0 0 -1.58-.74h-.43a2.06 2.06 0 0 0 -2.06 2.06v13.89a2.08 2.08 0 0 0 .26 1z" fill="#24272a"/></svg>
```
Comment (frol) [inline frontend/public/static/images/icon-sidebar.svg:1]: remove the unnecessary file

**Review by frol — APPROVED** [review]

**Pull request merged**